VISION.md

```diff
@@ -0,0 +1,9 @@
+# Vision
+
+Ebury Chameleon is developed thanks to the frontend expertise of the [Online team](https://fxsolutions.atlassian.net/wiki/spaces/TEAM/pages/119543461/ONL) members, Chameleon brings the components that Eburyonline needs to build the best online experience for the Ebury products.
```

Comment:
Ebury labs link maybe? This Atlassian link won't work outside of the organisation

Reply:
Minor thing, I prefer to keep this internal link.

Reply:
fair enough